charts/rocketchat/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: rocketchat
-version: 6.4.4
+version: 1.1.1
```

Comment:
Version goes down. The highest version is selected in Openshift and it does not have the values.schema.json file on that version.